base/libgit2/consts.jl

```diff
@@ -263,4 +263,12 @@ module Consts
                         CREDTYPE_SSH_INTERACTIVE    = Cuint(1 << 4),
                         CREDTYPE_USERNAME           = Cuint(1 << 5),
                         CREDTYPE_SSH_MEMORY         = Cuint(1 << 6))
+
+    @enum(GIT_CONFIG, CONFIG_LEVEL_DEFAULT = 0,
```

Comment:
I wasn't really suggesting that.  If you are going to write inline code documentation for enum values, why not get docstrings as well for free?